internal/compose/compose.go

```diff
@@ -247,6 +250,63 @@ func (p *Project) Logs(opts CommandOptions) ([]byte, error) {
 	return b.Bytes(), nil
 }
 
+// WaitForHealthy method waits until all containers are healthy.
+func (p *Project) WaitForHealthy(opts CommandOptions) error {
+	// Read container IDs
+	args := p.baseArgs()
+	args = append(args, "ps")
+	args = append(args, "-q")
+
+	var b bytes.Buffer
+	if err := p.runDockerComposeCmd(dockerComposeOptions{args: args, env: opts.Env, stdout: &b}); err != nil {
+		return err
+	}
+
+	containerIDs := strings.Split(strings.TrimSpace(b.String()), "\n")
+	for {
+		if signal.SIGINT() {
+			return errors.New("SIGINT: cancel waiting for policy assigned")
+		}
```

Comment:
Other place where we may want some context or channel for cancelation :slightly_smiling_face:

Reply:
Agree, another place to be covered while refactoring this runner.